src/js/popup/popup.js

```diff
@@ -343,10 +343,12 @@
             }, false);
           
           // Get masks and determine what to display
-          const masks = await popup.utilities.getMasks(getMasksOptions);
+          const { relayAddresses } = await browser.storage.local.get(
+            "relayAddresses"
+          );
```

Comment:
Context: This was one logic point where where we switched to local storage instead of waiting for the API results.

Reply:
Revised this placement in dffd2674682978a1b97e460c2b67caf308196259. 

However, we have still reduced the extra call(s) to the API, so the comment still stands.